File: src/formatValue.ts
export default function formatValue(value: any, multiline: boolean = true): string {
  if (typeof value !== 'string') {
    return '';
  }
  if (multiline) {
    return value.replace(/\\n/g, '\n');
  } else {
    return value.replace(/\n/g, '\\n');
  }
}

File: src/format_value.rs
/// Normalise newlines for output.
///
/// multiline=false (default): convert actual newlines → literal `\n` sequences.
/// multiline=true  (--multiline flag): expand literal `\n` sequences → actual newlines.
pub fn format_value(value: &str, multiline: bool) -> String {
    if multiline {
        value.replace("\\n", "\n")
    } else {
        value.replace('\n', "\\n")
    }
}
